### ISSUE TITLE
Idea for a better name for this extension

### ISSUE THREAD

**Issue by feren** [lestcape/Classic-Gnome#18]
For me, Classic GNOME sounds too generic and sounds like something that isn't Cinnamon on GNOME, so, I thought of a better name idea for it:

CinnaGnome maybe?

**Comment by bmeznarsic** [comment]
spicy-gnome

**Comment by lestcape** [comment]
Yes maybe, but i have not intention to promote Mint, Cinnamon or Spice... The intentions it's promote the standard user interface like occurs on Cinnamon and also the work of third-patry spices devs outside the Mint, Cinnamon and Spices scope.

**Comment by cinnamoner** [comment]
"Classic Gnome" comes closest to the concept. This is a non-issue.
